static/install

```diff
@@ -22,12 +24,21 @@ else
 fi
 
 echo "Installing MESG..."
+
+# fetch the latest version of mesg.
 link=$(curl -s https://api.github.com/repos/mesg-foundation/core/releases/latest | grep "browser_download_url.*$system.*$machine" | cut -d : -f 2,3 | tr -d \")
 
 path="/usr/local/bin/mesg-core"
 
+# install mesg-core binary.
 sudo curl --progress -L $link -o $path
 sudo chmod +x $path
 
+# install system services.
+git clone git@github.com:mesg-foundation/core.git /tmp/mesg-core
+mkdir -p ~/.mesg/systemservices
+cp -a /tmp/mesg-core/systemservices/_sources/. ~/.mesg/systemservices
```

Comment:
what's the `-a` for ?

Reply:
quoted from internet: `...basically it copies a directory exactly as it is; the files retain all their attributes...`

I though keeping it is more safe for this reason.

Reply:
ok fine, i never used that option before :)